meta-resin-morty/recipes-core/dropbear/files/atomic-hostkey.patch

```diff
@@ -0,0 +1,43 @@
+dropbearkey.c: Make host key generation atomic
```

Comment:
It seems like another solution ended up upstream. Can we use that? https://github.com/mkj/dropbear/pull/36